tests/playgrounds/screenshot.js

```diff
@@ -95,7 +95,7 @@ function workspaceToSvg_(workspace, callback, customCss) {
  * Download a screenshot of the blocks on a Blockly workspace.
  * @param {!Blockly.WorkspaceSvg} workspace The Blockly workspace.
  */
-Blockly.downloadScreenshot = function(workspace) {
+const downloadScreenshot = function(workspace) {
```

Comment:
Could you please add an eslint-ignore statement to silence this warning?

Reply:
Just converted it to a function declaration.